src/js/utils/utils.js

```diff
@@ -0,0 +1,16 @@
+export function createRandomNumber(maxNumber = 1) {
+  return Math.floor(Math.random() * maxNumber);
+}
+
+export function checkIsSameElementInArray(array) {
+  const set = new Set();
+
+  return array.some((el) => {
+    if (set.has(el)) {
+      return true;
+    }
+
+    set.add(el);
+    return false;
+  });
+}
```

Comment:
```suggestion
export function checkIsSameElementInArray(array) {  
  return new Set(array).size !== array.length
}
```

요렇게 간단하게 표현해볼 수 있을 것 같아요

Reply:
오! 감사합니다!